frontend/.agents/skills/resend/references/receiving.md

```
@@ -0,0 +1,287 @@
+**Critical:** Your MX record must have the lowest priority value, or emails won't route to Resend.
+
+### Subdomain Recommendation
+
+If you already have MX records (e.g., Google Workspace, Microsoft 365):
+
+| Approach | Result |
+|----------|--------|
+| **Use subdomain** (recommended) | `support.acme.com` → Resend, `acme.com` → existing provider |
+| **Use root domain** | All email routes to Resend (breaks existing email) |
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)